### ISSUE TITLE
how to train at FashionIQ?

### ISSUE THREAD

**Issue by Mrshenshen** [naver/artemis#1]
Hi, I would like to know if training on fashioniq requires extra operations, because my training effect is very poor, probably the score will not improve from 7epoch. During the training, vocab.py training shows missing 300 words, nothing else is abnormal,I just run train.py directly not to do extra changes.

**Comment by g-delmas** [comment]
Hi,

To reproduce the results from the paper, please note that you should actually run train.py twice (with a small change of arguments, as specified in ./scripts/release_script.sh). Indeed, before training the whole model end-to-end, there is a warm-up phase during which the pretrained weights from the CNN & the GloVe word embeddings are kept frozen (only the bi-gru/lstm and the ARTEMIS modules are trained). Instead of using train.py directly, you can use ./scripts/release_script.sh as indicated in the README:

`sh ./scripts/release_script.sh train ARTEMIS fashionIQ`

**_Note_**: it is normal to have 300 missing words (with respect to GloVe vocabulary) for FashionIQ. This is mainly due to some spelling mistakes in the modifying texts.
